src/python/module/smartredis/dataset_utils.py

```diff
@@ -50,6 +76,7 @@ def typecheck_stringlist(names, strings_name, string_name):
 
 class DatasetConverter:
     @staticmethod
+    @_requires_xarray
```

Comment:
This shouldn't need `xarray` as it's all SmartRedis natives.

Reply:
Bring in @billschereriii on this, as this was a recommendation from him

Reply:
reverting